compiler/rustc_codegen_llvm/src/declare.rs

```diff
@@ -90,7 +90,7 @@ impl CodegenCx<'ll, 'tcx> {
             name,
             fn_abi.llvm_cconv(),
             llvm::UnnamedAddr::Global,
-            fn_abi.llvm_type(self),
+            fn_abi.llvm_type(self, false),
```

Comment:
Huh, why is this setting the `decl` argument to `false` here? I would expect this to be set to true precisely because this is declaring a function, not necessarily defining it?

Reply:
It may just be that `decl` is a bad name for this -- see the other comment for why I introduced it at all. But basically, I left this and other callers `false` so that I didn't rock the boat, only changing behavior in my new targeted case.

Reply:
I've now removed that flag.